scripts/d4xx_to_mavlink.py

```diff
@@ -185,51 +187,76 @@
 obstacle_distance_msg_hz = args.obstacle_distance_msg_hz
 debug_enable = args.debug_enable
 
+def progress(string):
+    print(string, file=sys.stdout)
+    sys.stdout.flush()
+
 # Using default values if no specified inputs
 if not connection_string:
     connection_string = connection_string_default
-    print("INFO: Using default connection_string", connection_string)
+    progress("INFO: Using default connection_string %s" % connection_string)
 else:
-    print("INFO: Using connection_string", connection_string)
+    progress("INFO: Using connection_string %s" % connection_string)
 
 if not connection_baudrate:
     connection_baudrate = connection_baudrate_default
-    print("INFO: Using default connection_baudrate", connection_baudrate)
+    progress("INFO: Using default connection_baudrate %s" % connection_baudrate)
 else:
-    print("INFO: Using connection_baudrate", connection_baudrate)
+    progress("INFO: Using connection_baudrate %s" % connection_baudrate)
     
 if not obstacle_distance_msg_hz:
     obstacle_distance_msg_hz = obstacle_distance_msg_hz_default
-    print("INFO: Using default obstacle_distance_msg_hz", obstacle_distance_msg_hz)
+    progress("INFO: Using default obstacle_distance_msg_hz %s" % obstacle_distance_msg_hz)
 else:
-    print("INFO: Using obstacle_distance_msg_hz", obstacle_distance_msg_hz)
+    progress("INFO: Using obstacle_distance_msg_hz %s" % obstacle_distance_msg_hz)
 
 # The list of filters to be applied on the depth image
 for i in range(len(filters)):
     if filters[i][0] is True:
-        print("INFO: Applying: ", filters[i][1])
+        progress("INFO: Applying: %s" % filters[i][1])
     else:
-        print("INFO: NOT applying: ", filters[i][1])
+        progress("INFO: NOT applying: %s" % filters[i][1])
 
 if not debug_enable:
     debug_enable = debug_enable_default
 
 if debug_enable == 1:
-    print("INFO: Debugging option enabled")
+    progress("INFO: Debugging option enabled")
 else:
-    print("INFO: Debugging option DISABLED")
+    progress("INFO: Debugging option DISABLED")
 
 ######################################################
 ##  Functions - MAVLink                             ##
 ######################################################
 
+def MAVLinkLoop(conn, callbacks):
```

Comment:
Looks like we're switching to camel case when the rest of the script uses underlines?